msteams-platform/toolkit/explore-Teams-Toolkit.md

```diff
@@ -54,6 +54,7 @@ From the task pane, you can see the following sections:
 :::row:::
    :::column span="":::
       :::image type="content" source="../assets/images/teams-toolkit-v2/teams toolkit fundamentals/accounts1_1.png" alt-text="Screenshot shows the Accounts section.":::
+      :::image type="content" source="../assets/images/teams-toolkit-v2/teams toolkit fundamentals/switchTenants.png" alt-text="Screenshot shows the switch tenant button.":::
```

Comment:
Why this screenshot didn't show the tenant info?

Reply:
I hide the tenant info since it's my test account, do you think it's needed?